### PR TITLE
[W5.7][W16-4] LAW YONG QIANG CLINTON

### PR DESCRIPTION
**Pull request by clawyq**
Update user guide to reflect the change in the find method

### CODE REVIEW

**Comment by ChesterSng** [comment]
Very useful change to ab2, normally people wouldn't want to care about the upper/lower casing

**Comment by blewjy** [comment]
Good and practical feature!

**Comment by syahiranrafi** [comment]
Very useful feature!

**Comment by syahiranrafi** [comment]
Very useful feature!

**Comment by 99percentile** [comment]
Nice!! Now we do not need to care about caps when finding things

File: src/seedu/addressbook/commands/FindCommand.java [hunk 1]
```diff
@@ -41,16 +41,20 @@ public CommandResult execute() {
...
-     * Retrieves all persons in the address book whose names contain some of the specified keywords.
-     *
+     * Retrieves all persons in the address book whose names contain
+     * some of the specified keywords, regardless of casing.
      * @param keywords for searching
      * @return list of persons found
      */
     private List<ReadOnlyPerson> getPersonsWithNameContainingAnyKeyword(Set<String> keywords) {
+        ArrayList<String> lowerKeywords = new ArrayList<>();
+        for (String word : keywords) {
```
Comment (bqnguyen94) [inline src/seedu/addressbook/commands/FindCommand.java:51]: FYI: There's a neater way to do this. [See here.](https://dzone.com/articles/how-to-use-map-filter-collect-of-stream-in-java-8)

File: test/java/seedu/addressbook/commands/FindCommandTest.java [hunk 2]
```diff
@@ -36,7 +36,7 @@ public void execute() throws IllegalValueException {
                 Arrays.asList(td.amy, td.bill, td.candy));
 
         //repeated keywords: matched
-        assertFindCommandBehavior(new String[]{"Amy", "Amy"}, Arrays.asList(td.amy));
+        assertFindCommandBehavior(new String[]{"AMy", "AmY"}, Arrays.asList(td.amy));
```
Comment (bqnguyen94) [inline test/java/seedu/addressbook/commands/FindCommandTest.java:39]: I think `AMy` and `AmY` are not repeated keywords?

**Review by bqnguyen94 — CHANGES_REQUESTED** [review]
Good job 👍 just a minor change needed.

**Pull request closed without merging**